backend/tests/api/routes/test_ideas.py

```diff
@@ -916,3 +937,55 @@ async def test_POST_create_idea_returns_idea(
         assert data["upvoted_by"] == []
         assert data["downvoted_by"] == []
         assert data["creator_id"] == str(user.id)
+
+
+@pytest.mark.integration
+@pytest.mark.anyio
+async def test_POST_create_idea_creates_idea_in_db(
+    real_db: AIOSession,
+    user_with_client,
+):
+    fake = faker.Faker()
+    user, async_client = user_with_client
+
+    test_idea_create = {
+        "name": fake.sentence(nb_words=5, variable_nb_words=True),
+        "description": fake.paragraph(nb_sentences=5, variable_nb_sentences=True),
+    }
+
+    async with clean_new_ideas(real_db):
+        response = await async_client.post(
+            "/ideas/",
+            json=test_idea_create,
+        )
+
+        data = response.json()
+
+        assert response.status_code == 200
+
+        db_idea = await real_db.find_one(Idea, Idea.id == ObjectId(data["id"]))
+
+        assert db_idea.name == test_idea_create["name"]
+        assert db_idea.description == test_idea_create["description"]
+        assert db_idea.upvoted_by == []
+        assert db_idea.downvoted_by == []
+        assert db_idea.creator_id == user.id
+
+
+@pytest.mark.integration
+@pytest.mark.anyio
+@pytest.mark.parametrize(
+    "invalid_ideas",
+    INVALID_IDEAS,
+)
+async def test_POST_create_idea_returns_422_when_invalid_data(
+    real_db: AIOSession,
+    user_with_client,
+    invalid_ideas,
+):
+    user, async_client = user_with_client
```

Comment:
```suggestion
    _, async_client = user_with_client
```
If only second item of the unpacked `user_with_client` is used, the first one can be named with `_` to explicitly indicate it's not going to be used for anything.